echo-model/src/main/java/com/netflix/spinnaker/echo/model/Trigger.java

```diff
@@ -119,6 +125,9 @@ public String toString() {
   @JsonIgnore
   Pipeline parent;
 
+  @JsonIgnore
+  boolean propagateAuth;
```

Comment:
There's no fear of anyone being able to explicitly define this in their pipeline trigger json?

Reply:
This is a valid concern; the `@JsonIgnore` is not actually causing the field to be ignored in deserialization.  I'm going to figure out why that's happening, and ensure that any value in the JSON gets ignored.

Reply:
👍 

My preference would be to not have anything (externally settable) on the Trigger influence how auth is propagated.

Reply:
OK, configured the builder/deserializer to ignore that field, and added a test to ensure that `fropagateAuth` is always false after deserialization.